src/App/Header/NavDropdown.tsx

```diff
@@ -20,7 +25,7 @@ export default function NavDropdown({ links }: Props) {
     <Popover className="relative">
       {({ open }) => (
         <>
-          <Popover.Button
+          <PopoverButton
```

Comment:
most of the changes are of this nature
```suggestion
          <Popover.Button
```